modules/mapbox/src/mapbox-layer.ts

```diff
@@ -35,7 +35,9 @@ export default class MapboxLayer<LayerT extends Layer> implements CustomLayerInt
 
   onAdd(map: Map, gl: WebGLRenderingContext | WebGL2RenderingContext): void {
     if (gl instanceof WebGLRenderingContext) {
-      log.warn('Use useWebGL2:true in mapboxgl.Map()')();
+      log.warn(
```

Comment:
With MapboxLayer no longer exported, can the user get here without seeing the MapboxOverlay warning?

Reply:
They can't, so I can remove this